http-server/src/test/java/io/airlift/http/server/testing/AbstractTestTestingHttpServer.java

```diff
@@ -273,6 +276,47 @@ public void testGuiceInjectionWithResources()
         }
     }
 
+    @SuppressWarnings("SynchronizationOnLocalVariableOrMethodParameter")
```

Comment:
is this avoidable?

Reply:
We could move the method to the servlet and make it synchronized